src/om_bootstrap/nav.cljs

```diff
@@ -89,7 +89,9 @@
    [_]
    (let [{:keys [opts children]} (om/get-props owner)
          [bs props] (t/separate Nav opts {:bs-class "nav"})
-         classes {:navbar-collapse (:collapsible? bs)}
+         classes (d/class-set {:navbar-collapse (:collapsible? bs)
```

Comment:
okay, great, I can see this now that I'm on my computer.  Look at line 106 down below, where we call `(d/class-set classes)`. You need to remove the call to `d/class-set` up here.

Can you test that these changes work by running the docs project? I'm surprised that this would have worked at all with this bug.